lib/resolvers/place.js

```diff
@@ -1,18 +1,27 @@
 const { pick } = require('lodash');
 
-module.exports = ({ models }, { action, data, id }) => {
+module.exports = ({ models }) => ({ action, data, id }) => {
   const { Place } = models;
 
-  if (data.nacwo) {
-    data.nacwoId = data.nacwo;
-  }
+  if (data) {
+    if (data.nacwo) {
+      data.nacwoId = data.nacwo;
+    }
+    data.establishmentId = data.establishment;
 
-  data.establishmentId = data.establishment;
+    data = pick(data, 'id', 'site', 'area', 'name', 'suitability', 'holding', 'nacwoId', 'establishmentId');
```

Comment:
I know this has pulled straight over from the previous with indentation changes, but can this be refactored to do something with the defined keys in the json schema now?

Reply:
yes, I just didn't want to do too much here as it was supposed to be just adding tests